Library/Homebrew/exceptions.rb

```diff
@@ -495,6 +495,8 @@ def issues
 
   sig { returns(T::Array[T.untyped]) }
   def fetch_issues
+    return [] if ENV["HOMEBREW_NO_BUILD_ERROR_ISSUES"].present?
```

Comment:
```suggestion
    return [] if ENV["HOMEBREW_NO_EXCEPTIONS_FETCH_ISSUES"].present?
```

Reply:
This applies only for the `BuildError` exception, though, so `NO_BUILD_ERROR` seems more accurate to me.